### ISSUE TITLE
Last tick of direction getter returns -1

### ISSUE THREAD

**Issue by grcmichael** [darkroomengineering/lenis#63]
Hello,
I noticed a strange behaviour on the direction getter when I scroll down.
It works perfectly except when it stops. His last tick returns -1 instead of 1.

Here's my code : 
```
lenis.on('scroll', ({direction}) => {
  console.log(direction)
})
```

Here's a screenshot of my console
<img width="141" alt="Capture d’écran 2022-11-23 à 11 44 30" src="https://user-images.githubusercontent.com/54373418/203602527-905c81cb-8cbf-494a-8923-bd69dbac6131.png">

Am I missing something ?
Thank you very much :)

**Comment by joshmeads** [comment]
Opened a PR to resolve the issue. Unsure whether it should return 1 in that case or 0. Right now I have it set as 1.

**Comment by arzafran** [comment]
Thanks for the PR @joshmeads, but i took a slightly different approach and should be fixed in v0.2.24.

**Comment by grcmichael** [comment]
Thank you very much
